autoload/lsp/symbol.vim

```diff
@@ -811,33 +811,48 @@ def SymbolMenuFilterKey(symPopupMenu: number,
     # Erase a character in the input popup
     if inputText->len() >= 1
       inputText = inputText[: -2]
-      keyHandled = true
       updateInputPopup = true
+    else
+      keyHandled = false
     endif
   elseif key == "\<C-U>"
     # Erase all the characters in the input popup
     inputText = ''
-    keyHandled = true
     updateInputPopup = true
+  elseif key == "\<tab>"
+      || key == "\<C-n>"
+      || key == "\<Down>"
+      || key == "\<ScrollWheelDown>"
+    var ln = getcurpos(symPopupMenu)[1]
+    win_execute(symPopupMenu, "normal! j")
+    if ln == getcurpos(symPopupMenu)[1]
+      win_execute(symPopupMenu, "normal! gg")
+    endif
+  elseif key == "\<S-tab>"
+      || key == "\<C-p>"
+      || key == "\<Up>"
+      || key == "\<ScrollWheelUp>"
+    var ln = getcurpos(symPopupMenu)[1]
+    win_execute(symPopupMenu, "normal! k")
+    if ln == getcurpos(symPopupMenu)[1]
+      win_execute(symPopupMenu, "normal! G")
+    endif
+  elseif key == "\<PageDown>"
+    win_execute(symPopupMenu, 'normal! ' .. "\<C-d>")
+  elseif key == "\<PageUp>"
+    win_execute(symPopupMenu, 'normal! ' .. "\<C-u>")
```

Comment:
Can you use double quotes for the entire string and avoid the string concatenation ".."?